examples/deck.gl/pointcloud/app.js

```diff
@@ -113,25 +138,31 @@ export default class App extends PureComponent {
         // in which case the vertex count is not correct for display as points
         // Proposal: Consider adding a `mesh.points` or `mesh.pointcloud` option to mesh loaders
         // in which case the loader throws away indices and just return the vertices?
-        pointsCount: attributes.POSITION.value.length / 3, // header.vertexCount,
-        points: attributes.POSITION.value,
+        pointsCount: attributes.POSITION.value.length / attributes.POSITION.size,
```

Comment:
This is fine, though my preference would be the the application doesn't have to destructure the loaded data and store it in `pieces` like this.

Maybe just call the proposed `convertMeshToDeckPointCloudData` function here and store the `data` result?

Reply:
very good idea I've implemented this now in my latest commit